### PR TITLE
update rocksdb tuning for low IO.

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
bin/etc: rocksdb tuning(Low IO)

### CODE REVIEW

**Comment by zhangjinpeng1987** [comment]
PTAL @siddontang @ngaut @BusyJay @shenli

**Comment by siddontang** [comment]
LGTM

File: src/bin/tikv-server.rs [hunk 1]
```diff
@@ -167,20 +167,21 @@ fn get_rocksdb_option(matches: &Matches, config: &toml::Value) -> RocksdbOptions
                               Some("lz4".to_owned()),
                               |v| v.as_str().map(|s| s.to_owned()));
     let compression = util::rocksdb_option::get_compression_by_string(&tp);
-    let per_level_compression: [DBCompressionType; 7] = [DBCompressionType::DBNo,
+    opts.compression(compression);
+    /*let per_level_compression: [DBCompressionType; 7] = [DBCompressionType::DBNo,
```
Comment (shenli) [inline src/bin/tikv-server.rs]: Remove useless code.

**Comment by shenli** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Pull request merged**